tuned/consts.py

```diff
@@ -41,6 +41,13 @@
 KERNEL_UPDATE_HOOK_FILE = "/usr/lib/kernel/install.d/92-tuned.install"
 BLS_ENTRIES_PATH = "/boot/loader/entries"
 
+# scheduler plugin configuration
+# how many times retry to move tasks to parent cgroup on cgroup cleanup
+CGROUP_CLEANUP_TASKS_RETRY = 10
+PROCFS_MOUNT_POINT = "/proc"
+DEF_CGROUP_MOUNT_POINT = "/cgroup/cpuset"
```

Comment:
Well, I wasn't sure here. It's not FHS compliant, but it was mentioned in the older kernel docs. The /sys/fs/cgroup/cpuset would work great with cgroup v1 (e.g. on RHEL-8), because /sys/fs/cgroup is tmpfs mount point there. But it wouldn't work so great with the cgroup v2 enabled systemd (e.g. on Fedora when intermixing with cgroup v2). There /sys/fs/cgroup/ is cgroup2 FS, so the following would happen:
- cpuset group would be created after the mkdir call and all the special files would be auto populated by the kernel
- then we would mount over it the cgroup v1 FS, i.e. we would loose the previously created v2 group
In the end it would work, but it is not nice.

However, we are primary targeting RHEL-8 with this feature so I will probably change the default to integrate easily with the RHEL-8.

Reply:
On RHEL-8 /sys/fs/cgroup is read-only, so "cgroup_init" has to be set to False, i.e. with the default settings we rely on systemd to preset it.